packages/cli/src/tools/config/__tests__/index-test.js

```diff
@@ -236,3 +236,19 @@ test('should not add default React Native config when one present', () => {
   const {commands} = loadConfig(DIR);
   expect(commands).toMatchSnapshot();
 });
+
+test('should skip packages that have invalid configuration', () => {
+  writeFiles(DIR, {
+    'node_modules/react-native/package.json': '{}',
+    'node_modules/react-native/react-native.config.js': `module.exports = {
+      invalidProperty: 5
+    }`,
+    'package.json': `{
+      "dependencies": {
+        "react-native": "0.0.1"
+      }
+    }`,
+  });
+  const {dependencies} = loadConfig(DIR);
+  expect(dependencies).toMatchSnapshot();
```

Comment:
can you also spy on `logger.warn` and assert on the message? :) or snapshot it

Reply:
Should that be a separate test for appropriate warning or here?

Reply:
same test, different `expect` call :)